Khan/live-editor
# Add new Incredibles images to Pixar in a Box content

_See [LP-3444](https://khanacademy.atlassian.net/secure/RapidBoard.jspa?rapidView=40&modal=detail&selectedIssue=LP-3444) for more context._

This PR does the following:

- Adds some new images to the `/images/pixar` directory for some upcoming PiaB content
- Updates `all-images.js`

Test plan:

- Run `python -m SimpleHTTPServer 1313`
- Visit http://localhost:1313/demos/simple/
- Enter the following into the PJS editor: `image(getImage("pixar/Incredibles_a_fill"), 0, 0, 300, 125);`
- Observe that one the new Pixar in a Box images is rendered in the output

Here's one of the new images working in the PJS editor:

![screenshot of the new image in live-editor](https://user-images.githubusercontent.com/2208769/41486335-f0bf5c50-7098-11e8-95e4-187dcf2862f3.png)
